install/tools/setupMeshEx.sh

```diff
@@ -131,23 +131,37 @@ function istio_provision_certs() {
 #
 # Expected to be run from the release directory (ie istio-0.2.8/ or istio/)
 function istioBootstrapVM() {
- local DESTINATION=${1}
-
- local SA=${2:-${SERVICE_ACCOUNT:-default}}
- local NS=${3:-${SERVICE_NAMESPACE:-}}
- echo "Making certs for service account $SA (namespace $NS)"
- istio_provision_certs $SA $NS
-
- # Copy deb, helper and config files
- istioCopy $DESTINATION \
-   kubedns \
-   *.pem \
-   cluster.env \
-   istio.VERSION \
-   install/tools/setupIstioVM.sh
-
- # Run the setup script.
- istioRun $DESTINATION "sudo bash -c -x ./setupIstioVM.sh"
+  local DESTINATION=${1}
```

Comment:
in general it's best to split whitespace/formatting from actual changes in 2 commits (if not 2 PRs) so it's easier to not miss a change in the noise

Reply:
Thats indeed good practice. Perhaps we should also have a linter on shell script too.